.env.example

```
@@ -1,12 +1,12 @@
-SECURITY_ENC_KEY_BASE64=
-SECURITY_HASH_PEPPER=
+SECURITY_ENC_KEY_BASE64=CvUwRHDMpb3YInNkbfZpb5EQDjzOXeptSBDZ4rh8E4I=
+SECURITY_HASH_PEPPER=l6WrJOYmY0JuxbStYlTwIpdSr57i0uAOyUvHaNdQKB0=
 LOCAL_MAIL_HOST=email-smtp.ap-northeast-2.amazonaws.com
 LOCAL_MAIL_PORT=587
-LOCAL_MAIL_USERNAME=
-LOCAL_MAIL_PASSWORD=
+LOCAL_MAIL_USERNAME=AKIA43MWKDEDBEAQUUNV
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOCAL_MAIL_USERNAME key should go before the SECURITY_ENC_KEY_BASE64 key

(UnorderedKey)